scripts/actions/mobskills/grating_tantara.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- Grating Tantara
+--
+-- Description: Inflicts amnesia in an area of effect and deals damage.
+-- Type: Enfeebling
+-- Utsusemi/Blink absorb: Strips shadows
+-- Range: 10' as well as single target outside of 10'
+-- Notes: Doesn't use this if its horn is broken. Used only by certain Imp NMs, in place of Abrasive Tantara.
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    if mob:getAnimationSub() == 1 and mob:getFamily() == 165 then -- Imps without horn
+        return 1
+    else
+        return 0
+    end
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
+    local accmod = 1
+    local dmgmod = 3.0
+    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
```

Comment:
Do we want to set the dmgmod to 1 and put the ftp100, 200, 300 instead of this? I know we had talks of removing the dmgmod completely eventually because its a made up thing since ftp is implemented correctly.

`mtp000, mtp150, mtp300` in the function itself.
`xi.mobskills.mobPhysicalMove = function(mob, target, skill, numHits, accMod, dmgMod, tpEffect, mtp000, mtp150, mtp300, offcratiomod)`